### DISCUSSION TITLE
Add Fuzz testing to AUnit

### DISCUSSION THREAD

**Discussion by cyborgdennett** [bxparks/AUnit#103] [Ideas]
Hello,

I would like to request a feature for fuzz testing.

Fuzz testing is popular in golang, with inbuilt fuzz available.

It would be a cool feature for AUnit.

**Comment by bxparks** [comment]
Fuzzing is probably very useful for testing certain types of code. But personally, I have not used it, and I don't have any plans to support it in AUnit. It is not interesting enough or important enough to me to prioritize that above other things that I want to do.

I think the best course of action is for you to create a separate library to implement this. I don't want to incorporate it into AUnit because I don't want to be on the hook to provide support for it.
